src/specs/api.v2.yaml

```diff
@@ -100,6 +100,13 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HTTPError'
+        '424':
```

Comment:
I think I would have just reused another code like `403` to avoid having to add an extra response code to all of our endpoints. We could just use the `message` or `type` in the response to provide more info.

Reply:
I first tried reusing 403 but then I would have needed to add more checks in the UI on the message to check which kind of error it was and it seemed like just using a a different http code fits better with how we decide what to show in the UI.

For example, we show one [Result](https://ant.design/components/result/) when we got a 403 for an experiment, one when we got a 404 and this new one seemed to fit with using a differnet status codes